packages/app/src/app/components/Preview/DevTools/Terminal/ShellTabs/index.tsx

```diff
@@ -39,6 +41,9 @@ export default class ShellTabs extends React.PureComponent<Props> {
             />
           </Tab>
         ))}
+        <PlusContainer onClick={() => createShell()}>
```

Comment:
@CompuIves This can be simplified to just

```suggestion
        <PlusContainer onClick={createShell}>
```

Reply:
`createShell` can accept a string for the tab name, this causes the tab to open as "[Object object]" because the event is passed if it's called that way.

Reply:
OK this should be good then indeed 🙂